### ISSUE TITLE
DevTools Error: Parsing problem ☂️ bug

### ISSUE THREAD

**Issue by gkatsanos** [GoogleChrome/lighthouse#3306]
**Initial URL**: https://www.staging-lieferheld.de/
**Chrome Version**: 62.0.3202.9
**Error Message**: Parsing problem
**Stack Trace**:
```
Error: Parsing problem
    at Object.loadingComplete (chrome-devtools://devtools/remote/serve_file/@af8baa4f5884a58cb31839507cce2a4a5577f6c6/audits2_worker/audits2_worker_module.js:797:479)
    at WebInspector.TimelineLoader.cancel (chrome-devtools://devtools/remote/serve_file/@af8baa4f5884a58cb31839507cce2a4a5577f6c6/audits2_worker/audits2_worker_module.js:3335:37)
    at WebInspector.TimelineLoader._reportErrorAndCancelLoading (chrome-devtools://devtools/remote/serve_file/@af8baa4f5884a58cb31839507cce2a4a5577f6c6/audits2_worker/audits2_worker_module.js:3355:42)
    at WebInspector.TimelineLoader._writeBalancedJSON (chrome-devtools://devtools/remote/serve_file/@af8baa4f5884a58cb31839507cce2a4a5577f6c6/audits2_worker/audits2_worker_module.js:3353:49)
    at WebInspector.TextUtils.BalancedJSONTokenizer._reportBalanced (chrome-devtools://devtools/remote/serve_file/@af8baa4f5884a58cb31839507cce2a4a5577f6c6/audits2_worker/audits2_worker_module.js:2201:13)
    at WebInspector.TextUtils.BalancedJSONTokenizer.write (chrome-devtools://devtools/remote/serve_file/@af8baa4f5884a58cb31839507cce2a4a5577f6c6/audits2_worker/audits2_worker_module.js:2199:24)
    at WebInspector.TimelineLoader.write (chrome-devtools://devtools/remote/serve_file/@af8baa4f5884a58cb31839507cce2a4a5577f6c6/audits2_worker/audits2_worker_module.js:3347:31)
    at TraceParser.parseChunk (chrome-devtools://devtools/remote/serve_file/@af8baa4f5884a58cb31839507cce2a4a5577f6c6/audits2_worker/audits2_worker_module.js:800:30)
    at onChunkRead (chrome-devtools://devtools/remote/serve_file/@af8baa4f5884a58cb31839507cce2a4a5577f6c6/audits2_worker/audits2_worker_module.js:635:8)
    at <anonymous>
```

**Comment by patrickhulce** [comment]
Hmmm this is a new one... the public access to that URL is just the default 403 forbidden, is there anything special about the page and does this happen every time for you @gkatsanos?

**Comment by paulirish** [comment]
Interesting!

Unfortunately we lost an error message that was emitted around this time.

here are the possibilities:

* `"Malformed timeline data: Unknown JSON format"` - JSON didnt' start with { or [.. it's out of order
* `"Malformed timeline input, wrong JSON brackets balance"` - JSON didn't end right
* `"Malformed timeline data: %s", e.toString())` - a chunk of json coming from IO.read isn't valid despite the balancing. wouldn't expect this.
* `"Legacy Timeline format is not supported."` - seems weird.
* `"File \"%s\" not found.", reader.fileName()` - backend reports `NotFoundError`
* `"File \"%s\" is not readable", reader.fileName()` - backend reports `NotReadableError`
* `"An error occurred while reading the file \"%s\"", reader.fileName()` - backend reports some other error.

**Comment by gkatsanos** [comment]
The website is a staging environment / only accessible behind a specific corporate VPN, and the error didn't appear on a subsequent run. (so it was occasional)

**Comment by brendankenny** [comment]
This may have been fixed by #6056 (once it reaches DevTools stable)

**Comment by paulirish** [comment]
Chrome 71 is now stable. If you see this then you need to upgrade your Chrome!
